<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>19</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="tokenLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Token : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tokenLineEdit">
         <property name="text">
          <string>478be927679f6af89ae905e3b9dc710b</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="choixVilleLabel">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Ville : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="choixVilleCombo">
         <item>
          <property name="text">
           <string>Clermont-Ferrand</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Lyon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Paris</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Agde</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Miami</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="meteLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Météo : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="meteoLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="descLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Description : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="tempLabel">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Température : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="tempSpinBox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> °C</string>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="pressionLabel">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Pression : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="pressionSpinBox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> hPa</string>
         </property>
         <property name="maximum">
          <double>200000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="humLabel">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Humidité : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="humSpinBox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>choixVilleCombo</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>choixVille()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>activated(int)</signal>
  <slot>choixVille()</slot>
 </slots>
</ui>
